heros/hero1.html.slim

```diff
@@ -0,0 +1 @@
+= partial('hero1.html.slim');
```

Comment:
Yep - we need the partials to be publicly available so we can pull them in
via the js if we are doing ab testing
On Thu, Mar 10, 2016 at 4:21 AM Paul Robinson <notifications@github.com>
wrote:

> In heros/hero1.html.slim
> <https://github.com/redhat-developer/developers.redhat.com/pull/912#discussion_r55652260>
> :
>
> > @@ -0,0 +1 @@
> > += partial('hero1.html.slim');
>
> OK, I see now. I think you need the partial for the initial HTML build of
> the page, and then you use this file in the swap function. Is that right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/912/files#r55652260>
> .
>